### ISSUE TITLE
No Audio at DD+ (e-ac3) Option is On

### ISSUE THREAD

**Issue by Kampfader** [xbmc/xbmc#23363]

**Comment by fritsch** [comment]
The weirdness is, that the user runs a non official Kodi with workarounds for broken amlogic bsp. That's where the reopening for the sink comes from.

Please reproduce with a default nightly build.

**Comment by Kampfader** [comment]
Is there an official kodi nightly with a different package name?
I have only one firetv cube, and it is now running kodi 20 "org.xbmc.kodi".
If I install an omega nightly now, I'll completely ruin my system.

**Comment by Kampfader** [comment]
I have two kodi on my system.

I think it's a shame that nightly have the same package name as the official ones. If nightly had another package name, I would of course prefer it.

**Comment by fritsch** [comment]
[quote]
Starting Kodi (21.0-ALPHA1 (20.90.101) Git:20230531-5357f22024-dirty). Platform: Android ARM 32-bit
[/quote]

You are not running an official release, you use a nightly with an unknown git revision that uses kodi's official name.

**Comment by fritsch** [comment]
Yeah: 2023-06-06 14:27:00.149 T:21811    info <general>: The executable running is: /data/app/net.kodinerds.maven.kodi21.firetv-o7YSj_Qef_R38i04IGzIkw==/lib/arm/libkodi.so

Maven whatever.

**Comment by CastagnaIT** [comment]
i think he want install 2 kodi's on same device but you cant do it, or you keep kodi 20 or 21, not both
otherwise you need to fork kodi github project and modify it to build kodi with different android package name

**Comment by Kampfader** [comment]
I've now bought a Firetv Stick and installed the last Nightly there. Just set up Netflix, set all the settings we see above and started the same title. I added the new logs to the beginning of the post.

**Comment by fritsch** [comment]
Is Dolby+ in this combination via inputstream on netflix working for anyone? 

On Audiosink side nothing has changed and it is identical to version 20.x. From the error picture it seems that the sink does not want to move forward at all, which can only mean bogus data is sent out.

**Comment by Kampfader** [comment]
first post updatet, new logs

Passthrough output device - AudioTrack (IEC), Kodi IEC packer (recommended)
only works on Kodi Nexus and only on the FireTV Cube. Both devices are connected directly to the TV

with Passthrough output device - AudioTrack (RAW), Android IEC packer
it works on both devices. 

I don't want to install Omega on the Fire TV Cube yet. Because it's still in alpha status.

**Comment by prounlocks** [comment]
In my case NVIDIA HDMI output to Samsung Q700 soundbar (no eARC) - sound playing, but its not atmos,  Soundbar doesnt show ATMOS LOGO like vlc does. PASSTHRU option ON/OFF doesnt do anything in KODI 20.3  and 21 beta 2  - i tried all settings - no results.  NO passthru working.  VLC works well with passthru.  I think KODI have a bug with HDMI sound cards
